velox/connectors/hive/tests/PartitionIdGeneratorTest.cpp

```diff
@@ -145,8 +145,32 @@ TEST_F(PartitionIdGeneratorTest, stableIdsMultipleKeys) {
   }
 }
 
+TEST_F(PartitionIdGeneratorTest, partitionKeysCaseSensitive) {
+  PartitionIdGenerator idGenerator(
+      ROW({"cc0", "Cc1", "Cc2"}, {BIGINT(), VARCHAR(), INTEGER()}),
+      {1, 2},
+      100,
+      pool(),
+      false);
+
+  const vector_size_t size = 1'000;
+  auto input = makeRowVector({
+      makeFlatVector<int64_t>(size, [](auto row) { return row; }),
+      makeFlatVector<StringView>(
+          size,
+          [](auto row) {
+            return StringView::makeInline(DATE()->toString(18000 + row % 3));
+          }),
+      makeFlatVector<int32_t>(size, [](auto row) { return row % 7; }),
+  });
+
+  raw_vector<uint64_t> firstTimeIds;
+  idGenerator.run(input, firstTimeIds);
+  EXPECT_EQ("Cc1=2019-04-15/Cc2=1", idGenerator.partitionName(1));
```

Comment:
What does 1 mean in partitionName(1)? Why 1?

Reply:
@mbasmanova 1 means the partition number. I have converted the string column into partition columns. In this case, there are two partitions: one for `Cc1 =  apple` and another for` Cc1 = orange`. I have verified and checked all the partition names.